dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/internal/SipContainerComponent.java

```diff
@@ -123,7 +127,12 @@ protected void activate(ComponentContext context, Map<String, Object> properties
 			if (c_logger.isErrorEnabled())
 				c_logger.error("error.initialize.sip.container");
 		}
-		
+		useNetty = (!ProductInfo.getBetaEdition())?false:MetatypeUtils.parseBoolean(
+			"sipContainer", USE_NETTY, properties.get(USE_NETTY), true);
+		if (c_logger.isTraceDebugEnabled() && useNetty) {
```

Comment:
I've added message for the beta=true but useNetty=false case.  It could also be determined in the trace when it shows the sipcontainer parameter values.